diffkemp/diffkemp.py

```diff
@@ -529,7 +497,8 @@ def text_indent(text, width):
     old_dir_abs_path = os.path.join(os.path.abspath(snapshot_dir_old), "")
     new_dir_abs_path = os.path.join(os.path.abspath(snapshot_dir_new), "")
 
-    if fun_result.kind == Result.Kind.NOT_EQUAL:
+    any_diff = any([x.diff for x in fun_result.inner.values()])
```

Comment:
This looks like a potentially expensive operation and should be done lazily. Either inline the `any` call into the condition below or define `any_diff` as a lambda.